index.js

```diff
@@ -100,36 +100,38 @@ function parse(options) {
 }
 
 /**
+* @param  {Object} 					 options [posthtml options]
 * @param  {Array | Function} plugins [array of plugins to apply or function, which will be called with from option]
 * @param  {String}           from    [path to the processing file]
 * @param  {Object} 					 content [posthtml tree to process]
 * @param  {Array}            prepend [array of plugins to process before plugins param]
 * @return {Object}                   [processed poshtml tree]
 */
-function processWithPostHtml(plugins, from, content, prepend) {
+function processWithPostHtml(options, plugins, from, content, prepend) {
```

Comment:
@Scrum maybe `options` here would be better renamed to `parser`?

Reply:
Ideally, you need to pass all the options, then they are also passed to the parser